.github/workflows/java_api.yml

```diff
@@ -45,28 +47,55 @@ jobs:
           export ONNXRUNTIME_DIR=${PWD}
           export LD_LIBRARY_PATH=$ONNXRUNTIME_DIR/lib:$LD_LIBRARY_PATH
           popd
-      - name: Install opencv
+      - name: Install OpenCV
         run: |
           sudo apt-get install libopencv-dev
-      - name: Build java class
+      - name: Install ant
+        run: |
+          sudo apt-get install ant
+          export ANT_HOME=/usr/share/ant
+          export PATH=${ANT_HOME}/bin:${PATH}
+      - name: Build Java class
         run: |
           pushd csrc/mmdeploy/apis/java
           javac mmdeploy/*.java
           popd
-          pushd demo/java
-          javac -classpath ../../csrc/mmdeploy/apis/java/ Utils.java
-          popd
-      - name: Install mmdeploy with onnxruntime backend and java api
+      - name: Install mmdeploy with onnxruntime backend and Java API
         run: |
           mkdir -p build && pushd build
           export LD_LIBRARY_PATH=/home/runner/work/mmdeploy/mmdeploy/ncnn/install/lib/:$LD_LIBRARY_PATH
           cmake -DMMDEPLOY_BUILD_SDK=ON -DMMDEPLOY_BUILD_SDK_JAVA_API=ON -DMMDEPLOY_TARGET_BACKENDS=ort -DMMDEPLOY_CODEBASES=all -DONNXRUNTIME_DIR=~/work/mmdeploy/mmdeploy/onnxruntime-linux-x64-1.8.1  ..
           make install
+          echo $PWD
+          find /home/runner/work/mmdeploy/mmdeploy/build/lib/libmmdeploy_java.so
```

Comment:
I think it's better to use $GITHUB_WORKSPACE instead of hard code `/home/runner/work`